pkgs/applications/science/electronics/fritzing/default.nix

```diff
@@ -1,41 +1,75 @@
-{ mkDerivation, stdenv, fetchpatch, fetchFromGitHub, qmake, pkgconfig
-, qtbase, qtsvg, qtserialport, boost, libgit2
+{ mkDerivation, stdenv, fetchgit, fetchFromGitHub, qmake, pkgconfig
+, qtbase, qtsvg, qttools, qtserialport, boost, libgit2_0_27
 }:
 
+let
+  # tag corresponding to a release, has no further relation
+  fritzingTag = "CD-498";
+  # SHA256 of the fritzing-parts HEAD on the master branch,
+  # which contains the latest stable parts definitions
+  partsSha = "e79a69765026f3fda8aab1b3e7a4952c28047a62";
+in
+
 mkDerivation rec {
   pname = "fritzing";
-  version = "0.9.3b";
+  version = "0.9.4";
 
   src = fetchFromGitHub {
     owner = "fritzing";
     repo = "fritzing-app";
-    rev = version;
-    sha256 = "0hpyc550xfhr6gmnc85nq60w00rm0ljm0y744dp0z88ikl04f4s3";
+    rev = fritzingTag;
+    sha256 = "0aljj2wbmm1vd64nhj6lh9qy856pd5avlgydsznya2vylyz20p34";
   };
 
-  parts = fetchFromGitHub {
-    owner = "fritzing";
-    repo = "fritzing-parts";
-    rev = version;
-    sha256 = "1d2v8k7p176j0lczx4vx9n9gbg3vw09n2c4b6w0wj5wqmifywhc1";
+  parts = fetchgit {
+    url = "https://github.com/fritzing/fritzing-parts.git";
+    branchName = "master";
```

Comment:
The default branch is named `develop`, which is also ahead of the `master` branch.

Reply:
But I think `develop` might have broken part descriptions. `master` is basically guaranteed to be stable/working.

Reply:
You are totally right. That's also what the README says. Sorry.. perhaps I should have read it more carefully.